Sources/CryptorRSA/CryptorRSAKey.swift

```diff
@@ -531,44 +531,58 @@ extension CryptorRSA {
     /// - Returns: 		A tuple containing the (`PrivateKey`, `PublicKey`) instances.
 	///
 	public class func makeKeyPair(_ keySize: RSAKey.KeySize) throws -> (PrivateKey, PublicKey) {
-		
-        #if os(Linux)
-			var pkey = EVP_PKEY_new()
-			let ctx = EVP_PKEY_CTX_new_id(EVP_PKEY_RSA, nil)
-			defer {
-				EVP_PKEY_CTX_free(ctx)
-			}
-			guard EVP_PKEY_keygen_init(ctx) == 1,
-				EVP_PKEY_CTX_ctrl(ctx, -1, EVP_PKEY_OP_KEYGEN, EVP_PKEY_CTRL_RSA_KEYGEN_BITS, Int32(keySize.bits), nil) == 1,
-				EVP_PKEY_keygen(ctx, &pkey) == 1
-				else {
-					EVP_PKEY_free(pkey)
-					throw Error(code: ERR_INIT_PK, reason: "Could not generate rsa pair for \(keySize.bits) bits")
-			}
-			let privKey = PrivateKey(with: .make(optional: pkey))
-			let publicPem = try RSAKey.getPEMString(reference: privKey.reference, keyType: .publicType, stripped: false)
-			let pubKey = try CryptorRSA.createPublicKey(withPEM: publicPem)
+#if os(Linux)
+        var pkey = EVP_PKEY_new()
+		let ctx = EVP_PKEY_CTX_new_id(EVP_PKEY_RSA, nil)
+		defer {
+			EVP_PKEY_CTX_free(ctx)
+		}
+		guard EVP_PKEY_keygen_init(ctx) == 1,
+			EVP_PKEY_CTX_ctrl(ctx, -1, EVP_PKEY_OP_KEYGEN, EVP_PKEY_CTRL_RSA_KEYGEN_BITS, Int32(keySize.bits), nil) == 1,
+			EVP_PKEY_keygen(ctx, &pkey) == 1
+			else {
+				EVP_PKEY_free(pkey)
+				throw Error(code: ERR_INIT_PK, reason: "Could not generate rsa pair for \(keySize.bits) bits")
+		}
+		let privKey = PrivateKey(with: .make(optional: pkey))
+		let publicPem = try RSAKey.getPEMString(reference: privKey.reference, keyType: .publicType, stripped: false)
+		let pubKey = try CryptorRSA.createPublicKey(withPEM: publicPem)
```

Comment:
Any chance you can exclude spacing changes?  It makes it hard to tell what's a real code change.

Reply:
Ohh yup...looks like the repo using tabs, but my xcode uses spaces... I'll try to get that cleaned up...